src/Uno.Extensions.Core/PlatformHelper.cs

```diff
@@ -74,21 +74,26 @@ private static void EnsureInitialized()
 				|| RuntimeInformation.IsOSPlatform(OSPlatform.Create("BROWSER"));
 
 			// If wasm, then can assume app isn't packaged, so skip this check
-			if (!IsWebAssembly)
+			if (!IsWebAssembly && RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Testing for windows is likely enough:

```suggestion
			if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```